docs/managing/plugins.md

```diff
@@ -16,42 +16,69 @@ For example, to install the latest `aws` plugin:
 $ steampipe plugin install aws
 ```
 
-This will download the latest aws plugin (the one with the `latest` tag) from the hub registry, and will set up a default connection named `aws`. 
-
+This will download the latest aws plugin from the hub registry, and will set up a default connection named `aws`.
 
 ### Installing a Specific Version
 To install a specific version, simply specify the version tag after the plugin name, separated by `@` or `:`
 
-For example, to install the 0.0.1 version of the aws plugin:
+For example, to install the 0.118.0 version of the aws plugin:
 ```
-$ steampipe plugin install aws@0.0.1
+$ steampipe plugin install aws@0.118.0
 ```
 
-This will download the aws plugin version 0.0.1 (the one with the `0.0.1` tag) from the hub registry.  Note that installing a specific version will NOT create a connection for you - you must create/edit a [connection](/docs/managing/connections) configuration file in order to use the plugin. 
+This will download the aws plugin version 0.118.0 (the one with the `0.118.0` tag) from the hub registry.  Note that installing a specific version will NOT create a connection for you - you must create/edit a [connection](/docs/managing/connections) configuration file in order to use the plugin. 
```

Comment:
Is this still true?  I know that originally we only copied over the spc file as-is but I had thought that we changed the behavior at some point?

Reply:
Hm, it doesn't create it if you have an existing config for the plugin - however if you have no config for the plugin it will create it.

Reply:
Updated to be a single note that informs user only first installation of a plugin will give connection file, additional versions require amending/creation of new files.